rules/PSR4/Rector/FileWithoutNamespace/NormalizeNamespaceByPSR4ComposerAutoloadRector.php

```diff
@@ -148,8 +149,7 @@ private function refactorFileWithoutNamespace(
             }
         }
 
-        $namespace = new Namespace_(new Name($expectedNamespace), $nodes);
-        $nodesWithStrictTypesThenNamespace[] = $namespace;
+        $nodesWithStrictTypesThenNamespace[] = new Namespace_(new Name($expectedNamespace), $nodes);
```

Comment:
`$namespace` is used on next node after `if` in line 157, so `ExprUsedInNextNodeAnalyzer->isUsed()` check should be used.

Reply:
I'll fix the CI first :). I'll ask you for review after :+1: